packages/opencensus-web-types/scripts/copy-types.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2018, OpenCensus Authors
```

Comment:
Minor: `s/2018/2019`.

Reply:
Done, thank! I really need to set up a snippet in my editor for the copyright block.